### PR TITLE
Modified: Roadmap QA/Github

### PR DESCRIPTION
**Pull request by omdhede**
Added GitHub action in QA/GitHub section.
Solving Issue #7300

(Can't create a new node below CI/CD Skillset as asked in issue)

### CODE REVIEW

File: package-lock.json
Comment (arikchakma) [inline package-lock.json:1]: Please remove this file.

File: src/data/roadmaps/qa/content/github@v4gjV1qolrFXdESrY9bzG.md [hunk 1]
```diff
@@ -6,7 +6,10 @@ Visit the following resources to learn more:
 
 - [@opensource@GitHub Website](https://github.com)
 - [@article@GitHub Documentation](https://docs.github.com/en/get-started/quickstart)
+- [@opensource@Github Actions Website](https://github.com/features/actions)
+- [@article@Github Actions Documentation](https://docs.github.com/en/actions)
 - [@video@What is GitHub?](https://www.youtube.com/watch?v=w3jLJU7DT5E)
+- [@video@What is GitHub Actions?](https://youtu.be/URmeTqglS58?si=a_Lgh1S272z7jqv3)
```
Comment (arikchakma) [inline src/data/roadmaps/qa/content/github@v4gjV1qolrFXdESrY9bzG.md:12]: It's about GitHub, not GitHub actions, so please be specific to the topic. I don't think we should add them here.

Comment (omdhede) [reply src/data/roadmaps/qa/content/github@v4gjV1qolrFXdESrY9bzG.md:12]: I discussed, on discord Dan told me to add it to GitHub itself!
Is there any chance to create a new node below CI/CD node?

Comment (arikchakma) [reply src/data/roadmaps/qa/content/github@v4gjV1qolrFXdESrY9bzG.md:12]: Yes, we will do that in the next iteration.

Comment (omdhede) [reply src/data/roadmaps/qa/content/github@v4gjV1qolrFXdESrY9bzG.md:12]: Ok great!
Pin me if I would have a chance to contribute after that!

**Pull request closed without merging**